src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+import React, { useMemo } from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
+
+import { SearchView as CommonSearchView } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/SearchView';
+import { console, analytics, object, navigate as spcNavigate, valueTransforms } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { TableRowToggleOpenButton } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/table-commons';
+import { LocalizedTime } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/LocalizedTime';
+
+import { columnExtensionMap } from './columnExtensionMap';
+import { Schemas } from './../util';
+
+import { transformedFacets } from './SearchView';
+
+
+const publicationColExtensionMap = _.extend({}, columnExtensionMap, {
+    // We override display_title for this view to be wider.
+    // And awesomer.
+    "display_title" : {
+        "widthMap" : { "sm" : 320, "md" : 520, "lg" : 670 },
+        'minColumnWidth' : 200,
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            return <PublicationSearchResultTitle {...props} {...{ result }} />;
+        }
+    },
+    "nonexistent_year_field" : {
+        "widthMap" : { "sm" : 60, "md" : 60, "lg" : 60 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { date_published = null } = result;
+            if (!date_published) return null;
+            return (
+                <span className="value">
+                    <LocalizedTime timestamp={date_published} formatType="date-year" />
+                </span>
+            );
+        }
+    },
+    "journal" : {
+        "widthMap" : { "sm" : 100, "md" : 120, "lg" : 150 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { journal = null } = result;
+            if (!journal) return null;
+            return (
+                <span className="value">
+                    <JournalTitle journal={journal} width={props.width} />
+                </span>
+            );
+        }
+    },
+    "number_of_experiment_sets" : {
+        "widthMap" : { "sm" : 50, "md" : 50, "lg" : 50 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { number_of_experiment_sets: numSets = null } = result;
+            if (numSets === null) return null;
+            return (
+                <span className="value">
+                    { valueTransforms.decorateNumberWithCommas(numSets) }
+                </span>
+            );
+        }
+    }
+});
+
+/**
+ * This is memoized into a couple of different functions - 1 per column (common colWidth between all cells in same col)
+ * Make sure to keep this in sync with CSS rules (line height, etc.)
```

Comment:
There's more "accurate" ways of measuring how many lines a text block takes up but not quite performant or simple. Using "6px = upper bound of char width" rough estimate (+ padding, etc) isn't accurate but is simple/performant.

Reply:
Alternatively could use monospace font :-P (jk)